server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -305,7 +305,9 @@ export class MetadataExtractionProcessor {
     } catch (err) {
       ``;
       // do nothing
-      console.log('Error in video metadata extraction', err);
+      this.logger.error(
+        `Error in video metadata extraction for asset ${asset.id} due to ${err} on file ${asset.originalPath}`,
```

Comment:
We should add `err?.stack`. Also can we standardize on `err` or `error`. I would prefer `error` personally.